ivy/container/random.py

```diff
@@ -58,3 +58,47 @@ def random_uniform(
             map_sequences,
             out=out,
         )
+
+
+    # randint
+    @staticmethod
+    def static_randint(
+        low: Union[int, ivy.Container] = 0.0,
+        high: Union[int, ivy.Container] = 1.0,
+        shape: Optional[Union[int, Tuple[int, ...], ivy.Container]] = None,
+        device: Optional[Union[ivy.Device, ivy.NativeDevice, ivy.Container]] = None,
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        return ContainerBase.multi_map_in_static_method(
```

Comment:
Please update the docstring and add the static method examples here, you can refer [this](https://github.com/unifyai/ivy/blob/master/ivy/container/elementwise.py#L82) docstring for the same.